.github/workflows/ci.yml

```diff
@@ -236,7 +232,7 @@ jobs:
             which pip
 
       - name: Compilation Cache
-        uses: hendrikmuhs/ccache-action@v1.2
+        uses: hendrikmuhs/ccache-action@v1.2.9
```

Comment:
Not sure how exactly the version pins work in github, but "@3" sticks to the latest 3.x version, so I was guessing that "@1.2" would do the same for "1.2.x". That might depend on the versioning scheme of the specific action, though…

Reply:
well, many of the warnings are gone so I guess it did something